app/models/mixins/cloud_tenant_mixin.rb

```diff
@@ -1,8 +1,18 @@
 module CloudTenantMixin
   extend ActiveSupport::Concern
 
-  # TODO(lpichler) add synchronization
+  OPENSTACK_CLOUD_PROVIDER = 'Openstack Cloud Provider'.freeze
```

Comment:
TBH, I dont see why this should be a mixin in the first place. I looks like its only included in CloudManager - so move that code into CloudManager :smile: 

Then you can interpolate the name with 
```ruby
[5] pry(main)> ManageIQ::Providers::Openstack::CloudManager.description
=> "OpenStack"
```